sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -51,19 +52,19 @@ public int getOffset() {
     }
 
     /**
-     * Get the text sentiment labels: POSITIVE, NEGATIVE, NEUTRAL.
+     * Get the text sentiment label: POSITIVE, NEGATIVE, or NEUTRAL. Sentence sentiment label could not be MIXED.
```

Comment:
NIT: not sure we want to specify that mixed is not an option. It could trigger the question, but why not?
instead of just not mentioning it